tendermint/src/public_key.rs

```diff
@@ -1,73 +1,73 @@
 //! Public keys used in Tendermint networks
 
+pub use ed25519_dalek::PublicKey as Ed25519PublicKey;
```

Comment:
```suggestion
pub use ed25519_dalek::PublicKey as Ed25519;
```

Can we drop the suffix as it is already encoded in the module name? So callers can use it as `public_key::Ed25519`.

Reply:
Sure, I can push up that change, but I'd suggest doing it to all of the types (`secret_key::Ed25519Keypair`, `signature::Ed25519Signature`) for consistency.